java/src/main/java/com/ibm/plugin/rules/detection/bc/BouncyCastleInfoMap.java

```diff
@@ -40,6 +40,9 @@ public class BouncyCastleInfoMap {
     public static class Info {
         @Nullable private String name;
 
+        /**
+         * @deprecated use the key of the map instead
+         */
```

Comment:
Add the `@Deprected` annotation similar to [here](https://github.com/IBM/sonar-cryptography/blob/main/engine/src/main/java/com/ibm/engine/model/context/CipherContext.java#L61-L70)